SDK-2514: Add description how to chnage the default branch (#1827)
spryker/spryker-docs: docs/scu/dev/change-default-branch.md

---
title: Change default branch
description: How to change the default branch in Spryker CI
template: concept-topic-template
---

> To change the *default* branch is Spryker CI, do the following:

Reviewer: ```suggestion
To change the *default* branch in Spryker CI, do the following:
```

> 1. In [Spryker CI](/docs/scu/dev/spryker-ci.html), go to *Projects*.

Author: @AlexSlawinski , on another similar pages with instructions inside of the same documentation section "Spryker Code Upgrader" we already use bold style (**). So for me personally it doesn't matter what text formatting we will use, I just think that it makes more sense to use the same styling on all pages inside the same documentation section. 

Example with styling on another pages: 

https://github.com/spryker/spryker-docs/blob/master/docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-github-managed-project.md?plain=1

> 1. In [Spryker CI](/docs/scu/dev/spryker-ci.html), go to **Projects**.

Reviewer: ```suggestion
1. In [Spryker CI](/docs/scu/dev/spryker-ci.html), go to *Projects*.
```

> 2. On the **Projects** page, select the **Spryker Upgrade Service** project.

Reviewer: ```suggestion
2. On the *Projects* page, select the *Spryker Upgrade Service* project.
```

![Spryker CI Projects](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-github-managed-project.md/spryker_ci_projects.png)

> 3. On the **Pipelines** page, select **Code** menu item.

Reviewer: ```suggestion
3. On the *Pipelines* page, select the *Code* menu item.
```

> 4. On the **Code** page, select **Branches** tab.

Reviewer: ```suggestion
4. On the *Code* page, select the *Branches* tab.
```

![Set branch as default](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/change-default-branch.md/set-default-branch.png)

> 5. Near the branch that you want to set as *default* click on "..." button and click on **Set as default branch** button.

Reviewer: ```suggestion
5.  In the branch that you want to set as *default*, click on the "..." button to open a dropdown menu, then click on the **Set as default branch** button.
```
